client/components/GalleryModule.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+const GalleryModule = (props) => (
+  <div id="gallery">
```

Comment:
I really like what you did hear. That module is A+!

Reply:
It's so clean